plugin/evm/config.go

```diff
@@ -122,6 +123,7 @@ type Config struct {
 	AllowMissingTries               bool    `json:"allow-missing-tries"`                // If enabled, warnings preventing an incomplete trie index are suppressed
 	PopulateMissingTries            *uint64 `json:"populate-missing-tries,omitempty"`   // Sets the starting point for re-populating missing tries. Disables re-generation if nil.
 	PopulateMissingTriesParallelism int     `json:"populate-missing-tries-parallelism"` // Number of concurrent readers to use when re-populating missing tries on startup.
+	PruneWarpDB                     bool    `json:"prune-warp-db-enabled"`              // Determines if the warpDb should be cleared when re-initialzing subnet-evm
```

Comment:
Should we add add a validation for the case `PruneWarpDB` was enabled but `Pruning` was not? If these are not related, can we rename it to something like `warp-db-cleanup-enabled` etc? lIMHO one can confuse this with `pruning` and try to enable both of them.

Reply:
These are two different types of pruning.

I think pruning is the right term here tbh